src/Command/SF1500/SF1500LookupCommand.php

```diff
@@ -0,0 +1,267 @@
+
+        $help = <<<HELP
+
+user-id:
+    a user id, e.g.
+    'ffdb7559-2ad3-4662-9fd4-d69859939b66'
+
+certificate:
+    The certificate option can be a file path to a PKCS #12 certificate or a url query string with options for getting the certificate from an Azure Key Vault, e.g. (newlines added for readability and proper url encoding skipped)
+
+    tenant_id=f1d3b517-acac-4902-ae36-aaa32ed814dd
+    &client_id=7d711706-4325-47a6-a8c4-b373615c2b75
+    &client_secret=6e8551bf-2bb0-46bc-b449-647720120495
+    &name=organisation
+    &secret=test
+    &version=bbcbd812-0d7c-420b-9b93-7318e3769578
+    
+    If using path option make sure the path is relative to project root to the certificate file, e.g.
+    '/app/src/Command/SF1500/certificate.p12'
+ 
+passphrase:
+    the passphrase for p12 certificate, i.e.
+    'XYZ'
+    defaults to the empty string if not provided
+
+sts-applies-to:
+    the service SAML token should grant access to, i.e. SF1500 Organisation
+    'http://stoettesystemerne.dk/service/organisation/3'
+
+authority-cvr:
+    the authority cvr for which certificate is granted, e.g. Aarhus Kommune
+    '55133018'
+
+production:
+    use --production to use production mode rather than test mode
+
+manager:
+    user --manager to get information on the manger of provided user-id
+
+HELP;
+
+        $this->setHelp($help);
+
+        // Index by option name.
+        $this->inputOptions = array_combine(array_map(fn (InputOption $option) => $option->getName(), $inputOptions), $inputOptions);
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $resolver = new OptionsResolver();
+        $this->configureOptions($resolver);
+        $options = $resolver->resolve(
+            array_filter(array_filter($input->getOptions()), fn ($name) => isset($this->inputOptions[$name]), ARRAY_FILTER_USE_KEY)
+        );
+
+        $userId = $options['user-id'];
+
+        $propertyAccessor = new PropertyAccessor();
+
+        $certificateLocator = $this->getCertificateLocator($options['certificate'], $options['passphrase']);
+
+        $soapClient = new SoapClient([
+            'cache_expiration_time' => 'tomorrow 7am',
+        ]);
+
+        $serviceOptions = [
+            'certificate_locator' => $certificateLocator,
+            'authority_cvr' => $options['authority-cvr'],
+            'sts_applies_to' => $options['sts-applies-to'],
+            'test_mode' => !$options['production'],
+        ];
+
+        $sf1514 = new SF1514($soapClient, $serviceOptions);
+
+        $sf1500XMLBuilder = new SF1500XMLBuilder();
+
+        $sf1500 = new SF1500($soapClient, $sf1514, $sf1500XMLBuilder, $propertyAccessor, $serviceOptions);
+
+        // Use a lookup on name as an indicator for whether user-id exists or not.
+        $name = $sf1500->getPersonName($userId);
+
+        if (!$name) {
+            $output->writeln(sprintf('Could not find any user with the id: %s', $userId));
```

Comment:
Remove the colon. Colon should be used if there is a sort of stop in the sentence, e.g. "Invalid user id: %s".

Reply:
Fixed.